### ISSUE TITLE
Make use of Markdown-It on place of Remarkup + allow Vue components

### ISSUE THREAD

**Issue by vintprox** [vuephab/vuephab#7]
`markdown-it` comes with a lot of community plugins, which makes Markdown editing a joy. Remarkup processor, on the other hand, is limiting to self-enclosed markup.

What's more, Vuephab may offer usage of Vue components in Phriction documents to enhance authoring experience with interactive elements.

**Comment by Rizzen59** [comment]
Just for information. 
When you store a phriction document with remarkup, it is processed and stored in remarkup format and html format. 
When you display the phriction page, phabricator display directly the html format. 

I think there is possible collision between interpretor and old vue will be not usable

**Comment by vintprox** [comment]
> I think there is possible collision between interpretor and old vue will be not usable

Correct, markup generated with remarkup processor easily loses other potentially needed features. And because of that, original must be processed again through vuephab, including remarkup features and references. IIRC Nuxt has some cache management, and it may be of use to process original once per change.
